### ISSUE TITLE
Angular 10 support

### ISSUE THREAD

**Issue by mitschmidt** [ChristianKohler/ng-resize-observer#1]
First of all: Thanks for this useful package.

We've been using it happily with Angular v10 for a a couple of months - the peer dependencies should be updated to reflect v10 compatibility.

Regards from Munich

**Comment by ChristianKohler** [comment]
Thank you @mitschmidt 

I've updated the peer deps. Could you update the package and check if the peer deps are reflected?

**Comment by wottpal** [comment]
It seems to still not work

```
npm WARN ng-resize-observer@1.0.1 requires a peer of tslib@^1.10.0 but none is installed. You must install peer dependencies yourself.
```

Angular 10 uses tslib@2

**Comment by wottpal** [comment]
Any plans on updating this dependency @ChristianKohler? Thanks in advance :)

**Comment by ChristianKohler** [comment]
Fixed and tested with v1.0.2. Thank you @wottpal for reporting the issue.

**Comment by wottpal** [comment]
Great, thanks!! 🥳
